db/migrate/20241023080620_create_users.rb

```diff
@@ -0,0 +1,17 @@
+class CreateUsers < ActiveRecord::Migration[7.2]
+  def up
```

Comment:
don't we need to do indexing for any column?

Reply:
Added bhai. Thank you.